cubic-server/protocol/serialization/add.hpp

```diff
@@ -21,6 +21,31 @@ namespace protocol
     {
         return addVarInt(out, static_cast<int32_t>(data));
     }
+
+    constexpr void addBlockEntity(std::vector<uint8_t> &out, const BlockEntity &data)
+    {
+        addByte(out, data.sectionCoordinate);
+        addShort(out, data.height);
+        addVarInt(out, data.type);
+        addNBT(out, data.data);
+    }
+
+    constexpr void addBlockEntities(std::vector<uint8_t> &out, const std::vector<BlockEntity> &data)
+    {
+        addVarInt(out, data.size());
+        for (auto &i : data)
+            addBlockEntity(out, i);
+    }
+
+    constexpr void addLightArray(std::vector<uint8_t> &out, const std::vector<std::array<uint8_t, 2048>> &data)
```

Comment:
Use your constant here instead of putting a magic number

Reply:
:facepalm: